h2o-core/src/main/java/hex/Model.java

```diff
@@ -33,6 +36,14 @@
  */
 public abstract class Model<M extends Model<M,P,O>, P extends Model.Parameters, O extends Model.Output> extends Lockable<M> {
 
+  public P _parms;   // TODO: move things around so that this can be protected
```

Comment:
:+1: for the field protection. I think there are not performance critical fields.